core/chains/terra/terratxm/txm.go

```diff
@@ -167,7 +172,12 @@ func (txm *Txm) sendMsgBatch() {
 }
 
 func (txm *Txm) sendMsgBatchFromAddress(gasPrice sdk.DecCoin, sender sdk.AccAddress, key terrakey.Key, msgs terra.Msgs) {
-	an, sn, err := txm.tc.Account(sender)
+	tc, err := txm.tc()
```

Comment:
interesting, so this will roughly round robin the simulating/broadcasting/confirming over all the nodes configured at the expense of a db read + new connection per batch send

Reply:
Yeah and we pick up node reconfiguration right away.